src/testRunner/compilerRunner.ts

```diff
@@ -658,11 +662,15 @@ class IsolatedDeclarationTest extends CompilerTestBase {
         );
     }
 
+    verifyDiffReason() {
+        if (this.isOutputMapEquivalent && this.isOutputEquivalent && this.isDiagnosticEquivalent) {
+            ts.Debug.assert(this.diffReason === undefined, "Should not have a diff reason if everything is equivalent");
+        }else {

```

Comment:
nit: please add a space in between `}` and `else`

Reply:
forgot to format. thanks.